firstvoices/backend/serializers/page_serializers.py

```diff
@@ -1,18 +1,26 @@
 from rest_framework import serializers
 
-from backend.models import SitePage
-from backend.serializers.base_serializers import SiteContentLinkedTitleSerializer
+from backend.models import Image, SitePage
+from backend.models.constants import Visibility
+from backend.models.media import Video
+from backend.models.widget import SiteWidget, SiteWidgetList
+from backend.serializers.base_serializers import (
+    CreateSiteContentVisibilitySerializerMixin,
+    SiteContentLinkedTitleSerializer,
+    UpdateSerializerMixin,
+)
 from backend.serializers.fields import SiteHyperlinkedIdentityField
 from backend.serializers.media_serializers import ImageSerializer, VideoSerializer
+from backend.serializers.validators import SameSite
 from backend.serializers.widget_serializers import SiteWidgetListSerializer
 
 
 class SitePageSerializer(SiteContentLinkedTitleSerializer):
     url = SiteHyperlinkedIdentityField(
-        view_name="api:sitepage-detail", lookup_field="slug"
+        view_name="api:sitepage-detail", lookup_field="slug", read_only=True
     )
-    slug = serializers.CharField(read_only=True)
-    visibility = serializers.CharField(read_only=True, source="get_visibility_display")
+    slug = serializers.CharField()
```

Comment:
Did you check this one with @caforbes ? Not sure we want to let teams edit the slug for their site

Reply:
Sarah is right, both the language site slugs and the page slugs should not be editable by teams. In the frontend forms it is explicitly said "you cannot edit this".

Reply:
Good catch, thanks! I will update the PR.